app_customers/lib/themes/app_button_styles.dart

```diff
@@ -33,7 +33,7 @@ ButtonStyle roundedBigButton(
 /// The small rounded button used to add items to the cart.
 final roundedSmallButton = ButtonStyle(
   elevation: MaterialStateProperty.all(5.0),
-  backgroundColor: MaterialStateProperty.all(AppColors.blue),
+  backgroundColor: MaterialStateProperty.all(AppColors.darkBlack),
```

Comment:
:smile: you mean darkGrey i presume. if not just say black.
Please rename the `AppColors.darkBlack` color name